### DISCUSSION TITLE
New menu items

### DISCUSSION THREAD

**Discussion by AliDehbansiahkarbon** [AliDehbansiahkarbon/ChatGPTWizard#36] [Ideas]
Guys, I will add some functions to the Editor using new menu items.
Please share with me if you have any suggestions in this regard.

![image](https://user-images.githubusercontent.com/5601608/233333245-2016d57a-28cf-40b3-904a-ad7e8ea0d090.png)

**Comment by mcirinei** [comment]
Hi,
I've just installed your plugin in Delphi 11.3 but I can't see any context menu item except first (Ask).
Any idea?
Thanks for your work.
Marco

**Reply by AliDehbansiahkarbon** [reply]
FYI: Getit's version has been updated.

**Comment by AliDehbansiahkarbon** [comment]
Hi, This is a new feature and it's not available on the Getit version yet.
You can get a clone or zip from this repository if you prefer to use the latest version.
The installation will be a bit different than Getit, please read the ReadMe, the installation section if you need it.

(https://github.com/AliDehbansiahkarbon/ChatGPTWizard#:~:text=How%20to%20Install,build%2C%20and%20install.)
